index.html

```diff
@@ -2,15 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
+    <title>Review Project</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
     <div class="container">
```

Comment:
Please the script should be on top of the body, for a better code. 👌

Reply:
- [OPTIONAL] This comment is marked optional but highly recommended to heed the instructions here for the benefit of future review. The changes you make reflect on the right side where I wrote _*Review should be done here.*_. That's where you have green highly not other way round. Please, take this instructions very seriously so that you would not make the same mistake when next you review someone's code.

![review-side](https://user-images.githubusercontent.com/26597432/193638023-0712784f-e5c7-493e-b985-a1ad3599d91f.png)